build.gradle

```diff
@@ -133,8 +133,8 @@ asciidoctor {
         idprefix: '',  // for compatibility with GitHub preview
         idseparator: '-',
         'site-root': "${sourceDir}",  // must be the same as sourceDir, do not modify
-        'site-name': 'AddressBook-Level3',
-        'site-githuburl': 'https://github.com/se-edu/addressbook-level3',
+        'site-name': 'weme',
+        'site-githuburl': 'https://github.com/AY1920S1-CS2103-T16-3/main',
         'site-seedu': true,  // delete this line if your project is not a fork (not a SE-EDU project)
```

Comment:
Should we delete this line too? If so, the corresponding lines in `docs/templates/_header.html.slim` can also be deleted.

Reply:
isnt our proj a fork

Reply:
See https://github.com/nus-cs2103-AY1920S1/forum/issues/128